### PR TITLE
(SIMP-6936) HOWTO for simp config env change

### PR DESCRIPTION
**Pull request by trevor-vaughan**
Added a HOWTO for using an alternate target environment during `simp
config` and `simp bootstrap`

SIMP-6936 #comment Added HOWTO for 'simp config' environment change

### CODE REVIEW

File: docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Generally, when running the ``simp config`` and ``simp bootstrap`` commands, all
+items will be placed into the ``production`` :term:`Puppet Environment` and
+:term:`SIMP Secondary Environment`.
+
+If you are installing SIMP onto a preexisting system, you may want to instead
+install into an environment other than ``production``.
+
+As of version 6.0.0, the ``simp-cli`` was updated to support setting an
+alternate initial environment using a shell environment variable called
+``SIMP_ENVIRONMENT``.
```
Comment (op-ct) [inline docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst:15]: The 6.0.0 here refers to the version of `simp-cli`, but in the context of the SIMP docs, it's more likely to be interpreted as a SIMP release.  Here's one way to clarify it:

```suggestion
As of ``simp-cli`` version 6.0.0 (new with SIMP 6.5), you can configure
the ``simp config`` and ``simp bootstrap`` commands to use an
alternate initial environment by setting the shell environment 
variable, ``$SIMP_ENVIRONMENT``.
```

File: docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.. _howto-use-an-alternate-simp-config-environment:
+
+HOWTO Use an Alternate ``simp config`` Environment
+==================================================
+
+Generally, when running the ``simp config`` and ``simp bootstrap`` commands, all
+items will be placed into the ``production`` :term:`Puppet Environment` and
```
[the remark this answers is not in the document]
Comment (trevor-vaughan) [reply docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst:7]: Yeah, probably but for another day :-D

File: docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.. _howto-use-an-alternate-simp-config-environment:
+
+HOWTO Use an Alternate ``simp config`` Environment
```
[the remark this answers is not in the document]
Comment (trevor-vaughan) [reply docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst:3]: Well, that's fun. I'll remove the backticks.

[the remark this answers is not in the document]
Comment (trevor-vaughan) [reply docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst:3]:
```suggestion
HOWTO Use an Alternate 'simp config' Environment
```

**Pull request merged**